src/MatrixClient.ts

```diff
@@ -844,8 +844,11 @@ export class MatrixClient extends EventEmitter {
                 }
                 if (event['type'] === 'm.room.message') {
                     await emitFn("room.message", roomId, event);
-                }
-                if (event['type'] === 'm.room.tombstone' && event['state_key'] === '') {
+                } if (event['type'] === 'm.reaction') {
+                    await emitFn("reaction", roomId, event);
+                } if (event['type'] === 'm.room.redaction') {
+                    await emitFn('room.redaction', roomId, event);
```

Comment:
these should be captured by `room.event`? Only frequently used ones (messages) or ones which have special behaviour (room upgrades) are exposed at the moment, rather than any type.

Reply:
Ah I see, that makes sense. I'll close this PR, thanks!